user/views.py

```diff
@@ -12,6 +15,72 @@ def check_login(request):
 
 
 def home(request):
-    if request.user.is_authenticated:        
+    if request.user.is_authenticated:
         return render(request,'tweet/home.html')
-    return redirect('index')
\ No newline at end of file
+    return redirect('index')
+
+def signup(request):
+    if request.user.is_authenticated:
+        return redirect('index')
```

Comment:
why not this ?
```suggestion
return redirect('index') if request.user.is_authenticated else None
```

Reply:
https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3

Reply:
done